zmon_worker_monitor/zmon_worker/notifications/push.py

```diff
@@ -23,12 +27,23 @@ def formatEntity(entity_id):
 class NotifyPush(BaseNotification):
     @classmethod
     def notify(cls, alert, *args, **kwargs):
-        url = kwargs.get('url', cls._config.get('notifications.push.url'))
-        key = kwargs.get('key', cls._config.get('notifications.push.key'))
+        url = kwargs.get('url', cls._config.get('notification.service.url', None))
+        key = kwargs.get('key', cls._config.get('notification.service.key', None))
```

Comment:
Why do we need this at all when we can just use `OAUTH2_ACCESS_TOKENS`? `oauth2` is assumed to be `True` by default anyway... (see next line)